File: src/Lesson5/java/Ex1.java
package Lesson5.java;

public class Ex1 {
    public static void main(String[] args) {

        int[] arr = {5, 2, 9, 1, 7};

        int max = arr[0];
        int min = arr[0];

        int maxIndex = 0;
        int minIndex = 0;

        for (int i = 0; i < arr.length; i++) {

            if (arr[i] > max) {
                max = arr[i];
                maxIndex = i;
            }

            if (arr[i] < min) {
                min = arr[i];
                minIndex = i;
            }
        }

        System.out.println("Max value: " + max + ", index: " + maxIndex);
        System.out.println("Min value: " + min + ", index: " + minIndex);
    }

}

File: src/Lesson5/forkotlin/Ex1.kt
package Lesson5.forkotlin

fun main() {
    val arr = intArrayOf(5, 2, 9, 1, 7)

    var max = arr[0]
    var min = arr[0]

    var maxIndex = 0
    var minIndex = 0

    for (i in arr.indices) {
        if (arr[i] > max) {
            max = arr[i]
            maxIndex = i
        }

        if (arr[i] < min) {
            min = arr[i]
            minIndex = i
        }
    }

    println("Max value: $max, index: $maxIndex")
    println("Min value: $min, index: $minIndex")
}